homeassistant/components/scrape/config_flow.py

```diff
@@ -95,26 +95,31 @@
     vol.Optional(CONF_ENCODING, default=DEFAULT_ENCODING): TextSelector(),
 }
 
+NONE_SENTINEL = "none"
```

Comment:
I feel that the sentinel should be an empty string, because when users want to clear the input I think they will find the empty string more easily than a sentinel (whether it's translated or not, whether it's at the bottom or the top).

Reply:
I think you're mixing up the value, which the user do not know they are selecting, with the label which they will see in the dropdown.
Regardless of if the value is `""` or `"None"` the option to select should be "No device class", and it should indeed be the first or last option.

Reply:
I added translations as requested.
It's a shame that we can't use leading/trailing underscore here as it might have made more sense for a sentinel value `"__none__"`

Reply:
Why can't we use leading underscore?